package/src/renderer/__tests__/e2e/AnimatedImages.spec.tsx

```diff
@@ -15,6 +15,7 @@ describe("Animated Images", () => {
       if (!animatedImage) {
         return false;
       }
+      animatedImage.getFrameCount();
```

Comment:
nice! maybe here instead of returning true you can return the frame count and then test its value in expect. I guess in this particular test it will be one.